maestro-xcuitest-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -145,71 +133,29 @@ class XCTestDriverClient(
         endY: Double,
         duration: Double
     ): Response {
-        val mediaType = "application/json; charset=utf-8".toMediaType()
-        val request = SwipeRequest(
+        return executeJsonRequest("swipe", SwipeRequest(
             startX = startX,
             startY = startY,
             endX = endX,
             endY = endY,
             duration = duration
-        )
-        val body = mapper.writeValueAsString(request).toRequestBody(mediaType)
-
-        val url = xctestAPIBuilder("swipe")
-            .addQueryParameter("appId", appId)
```

Comment:
Isn't this necessary anymore?

Reply:
Wipe does not need the appId argument indeed. See https://github.com/mobile-dev-inc/maestro/blob/main/maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/SwipeRouteHandler.swift